docs/UserGuide.md

```diff
@@ -29,7 +29,7 @@ the process of accessing and updating resident student details. What's more, Dor
     - [Adding a person : `add`](#adding-a-person-add)
     - [Listing all persons : `list`](#listing-all-persons--list)
     - [Editing a person : `edit`](#editing-a-person--edit)
-    - [Finding a person : `find`](#locating-persons-by-name-phone_number-room_number-or-tag-find)
+    - [Finding a person : `find`](#locating-persons-by-name-phone-room_number-or-tag-find)
```

Comment:
This leads to a broken hyperlink. Should it (#finding-a-person--find) instead? Or should we update the title of find?